src/api/TokenApi/tokenList.ts

```diff
@@ -0,0 +1,134 @@
+
+  // TrueUSD
+  //  https://www.trusttoken.com/trueusd
+  //  US Dollar backed stable coin which is totally fiat-collateralized
+  {
+    name: 'TrueUSD',
+    symbol: 'TUSD',
+    decimals: 18,
+    addressByNetwork: {
+      1: '0x0000000000085d4780B73119b644AE5ecd22b376',
+      4: '0x0000000000000000000000000000000000000000', // TODO: Find/Deploy and update?
+    },
+  },
+
+  // USD Coin
+  //  https://www.circle.com/en/usdc
+  //  legitimate built on open standards
+  //  using Grant Thornton for verifying Circle’s US dollar reserves
+  //  launched by cryptocurrency finance firm circle Internet financial Ltd and the CENTRE open source consortium launched
+  {
+    name: 'USD Coin',
+    symbol: 'USDC',
+    decimals: 6,
+    addressByNetwork: {
+      1: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+      4: '0x0000000000000000000000000000000000000000', // TODO: Find/Deploy and update?
+    },
+  },
+
+  // Paxos Standard (PAX)
+  //  https://www.paxos.com/standard
+  //  regulated stable coin. PAX is backed by US Dollar in equivalent 1:1.
+  //  launched by Paxos Trust Company
+  //  approved by the New York State Department of Financial Services
+  {
+    name: 'Paxos Standard',
+    symbol: 'PAX',
+    decimals: 18,
+    addressByNetwork: {
+      1: '0x8e870d67f660d95d5be530380d0ec0bd388289e1',
+      4: '0x0000000000000000000000000000000000000000', // TODO: Find/Deploy and update?
+    },
+  },
+
+  // Paxos Gemini
+  //  https://gemini.com/dollar/
+  //  regulated by the New York State Department of Financial Services
+  //  launched same day as PAX by Gemini Trust Company. backed by USD
+  {
+    name: 'Paxos Gemini',
+    symbol: 'GUSD',
+    decimals: 2,
+    addressByNetwork: {
+      1: '0x056Fd409E1d7A124BD7017459dFEa2F387b6d5Cd',
+      4: '0x0000000000000000000000000000000000000000', // TODO: Find/Deploy and update?
+    },
+  },
+
+  // MakerDAO (DAI)
+  //  https://makerdao.com/
+  //  crypto-collateralized cryptocurrency: stable coin which is pegged to USD
+  {
+    name: 'DAI Stablecoin',
+    symbol: 'DAI',
+    decimals: 18,
+    addressByNetwork: {
+      1: '0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359',
+      4: '0x0000000000000000000000000000000000000000', // TODO: Find/Deploy and update?
+    },
+  },
+]
+
+export function getTokensByNetwork(networkId: number): TokenDetails[] {
+  // Get mainnet address by token (required for examples to get the image)
+  const mainnetAddresses = tokens.reduce((acc, { symbol, addressByNetwork }) => {
+    const addressMainnet = addressByNetwork[1]
+    if (symbol && addressMainnet) {
+      acc[symbol] = addressMainnet
+    }
+    return acc
+  }, {})
+
+  // Return token details
+  const tokenDetails: (TokenDetails | null)[] = tokens.map(token => {
+    const address = token.addressByNetwork[networkId]
+    if (address) {
+      // There's an address for the current network
+      const { name, symbol, decimals } = token
+      const addressMainnet = mainnetAddresses[symbol]
```

Comment:
Why not just read token.addressByNetwork[1] here? Why do we need to reduce mainnet addresses separately?

Reply:
Probably because I was sleepy!

When I did it, i thought about it, but I thought I didn't have access to the `addressByNetwork` in this iteration.

Super good catch